src/modules/PublicLab.MainImageModule.js

```diff
@@ -147,27 +147,29 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
     var imageInput = document.getElementById('thumbnail-img');
     var infoArea = document.getElementById('thumbnail-filename');
 
-    imageInput.addEventListener('change', showFileName);
+    if (imageInput) {
```

Comment:
We should use `imageInput && infoArea` in if condition

Reply:
if we use `image input` in if condition then we cant check it. By the way its working fine now,

Reply:
Yes good to know it is working 👍
I am recommending this change: 
```suggestion
    if (imageInput && infoArea) {
```

Let's say `thumbnail-filename` id is not present, then using `infoArea` variable will break UI, right?

Reply:
oh ok I got it.